modules/cats-effect/src/test/scala/pureconfig/module/catseffect/CatsEffectSuite.scala

```diff
@@ -47,6 +48,22 @@ class CatsEffectSuite extends BaseSuite {
     thrown.failures.head shouldBe a[ConvertFailure]
   }
 
+  it should "run successfully from a Typesafe Config object" in {
```

Comment:
Can you add an example where the reader fails, both in `loadConfigF(Config)` and in `loadConfigF(Config, String)`?

Reply:
Sure!